### PR TITLE
Release/rc 12.55.32.1

### PR DESCRIPTION
**Pull request by paul-ciobanu**

### CODE REVIEW

File: OctaneVSPlugin/Common/EntityComparerByLastModified.cs [hunk 1]
```diff
@@ -18,45 +18,38 @@
...
-
-            if (isDescending)
+            if (!secondDateTime.HasValue)
             {
-                result = -1 * result;
+                return 1;
             }
 
-            return result;
+            return -1 * firstDateTime.Value.CompareTo(secondDateTime.Value);
```
Comment (kdrtibor) [inline OctaneVSPlugin/Common/EntityComparerByLastModified.cs]: Why is it needed to reverse the sign of the result of CompareTo?

Comment (paul-ciobanu) [reply OctaneVSPlugin/Common/EntityComparerByLastModified.cs]: @kdrtibor Because we want to sort by descending date

Comment (kdrtibor) [reply OctaneVSPlugin/Common/EntityComparerByLastModified.cs]: @paul-ciobanu  but why not switch only the order of comparison?

Comment (paul-ciobanu) [reply OctaneVSPlugin/Common/EntityComparerByLastModified.cs]: @kdrtibor good point. fixed

**Pull request merged**